apps/web/components/pages/task/description-block/editor-components/serializerConfigurations.ts

```diff
@@ -24,6 +25,22 @@ export const configHtmlToSlate: HtmlToSlateConfig = {
 		ol: () => ({ type: 'ol' }),
 		p: () => ({ type: 'p' }),
 		ul: () => ({ type: 'ul' }),
+		//@ts-ignore
+		div: (el) => {
+			if (el && el.attribs.class === 'checklist-container') {
+				const checkbox = el.firstChild as Element;
+				const textNode = el.lastChild as Element;
+				return {
+					type: 'checklist',
+					checked: checkbox && checkbox.attribs.checked === 'true',
+					children: [
+						//@ts-ignore
+						{ text: textNode && textNode.firstChild.data },
+					],
+				};
+			}
+			return undefined;
```

Comment:
@desperado1802 seriously, return undefined? :)

Reply:
It returns undefined when the condition is not met. If the div element from DB doesn't have class "checklist-container". That's the only type div element we have as editor html value anyway, for that reason it won't return undefined